### PR TITLE
Update query-builder.rst

### PR DESCRIPTION
**Pull request by celsowm**
tuple comparison examples

### CODE REVIEW

File: en/orm/query-builder.rst [hunk 1]
```diff
@@ -1208,6 +1208,56 @@ And can be used combined with aggregations too::
         ->innerJoinWith('Stocks')
         ->groupBy(['Products.id', 'Products.name', 'Products.unit_price']);
 
+Tuple Comparison
+----------------
+
+Tuple comparison involves comparing two rows of data (tuples) element by element,
+typically using comparison operators like `<, >, =`::
```
Comment (dereuromark) [inline en/orm/query-builder.rst]: You need double `` for code fencing.

Comment (dereuromark) [reply en/orm/query-builder.rst]:
```suggestion
typically using comparison operators like ``<, >, =``::
```
afaik

Also other examples that are inline text.

**Review by Arhell — APPROVED** [review]

File: en/orm/query-builder.rst [hunk 2]
```diff
@@ -1208,6 +1208,56 @@ And can be used combined with aggregations too::
...
+        ->where([
+            'OR' => [
+                ['unit_price <' => 20],
+                ['unit_price' => 20, 'tax_percentage <=' => 5],
+            ]
+        ]);
+   
+    # WHERE (unit_price < 20 OR (unit_price = 20 AND tax_percentage <= 5))
+
+The same result can be achieved using `TupleComparison`::
```
Comment (markstory) [inline en/orm/query-builder.rst]:
```suggestion
The same result can be achieved using ``TupleComparison``::
```

File: en/orm/query-builder.rst [hunk 3]
```diff
@@ -1208,6 +1208,56 @@ And can be used combined with aggregations too::
...
+                ['unit_price', 'tax_percentage'],
+                [20, 5],
+                ['integer', 'integer'], # type of each value
+                '<='
+            )
+        );
+
+    # WHERE (unit_price, tax_percentage) <= (20, 5))
+
+Tuple Comparison can also be used with `IN` and the result can be transformed
```
Comment (markstory) [inline en/orm/query-builder.rst]:
```suggestion
Tuple Comparison can also be used with ``IN`` and the result can be transformed 
```

File: en/orm/query-builder.rst [hunk 4]
```diff
@@ -1208,6 +1208,56 @@ And can be used combined with aggregations too::
...
+                    [[10, 10], [30, 10]],
+                    ['integer', 'integer'],
+                    'IN'
+                ),
+            );
+
+    # WHERE (1) = ( SELECT (1) WHERE ( ( articles.id = : 10 AND articles.author_id = : 10 ) OR ( articles.id = : 30 AND articles.author_id = : 30 ) ) )
+
+.. note::
+    Tuple comparison transform only supports the `IN` and `=` operators
```
Comment (markstory) [inline en/orm/query-builder.rst]:
```suggestion
    Tuple comparison transform only supports the ``IN`` and ``=`` operators
```

**Pull request merged**